CSS/Styles.css

```diff
@@ -380,9 +476,186 @@ textarea {
 .socialinks li {
   display: inline-block;
   padding: 0;
+  width: fit-content;
+  height: auto;
 }
 
 .socialinks img {
   height: 50%;
   width: auto;
 }
+
+@media screen and (min-width: 768px) {
+  .nav {
+    box-shadow: 0 0.5px 0 #dddfe6;
+    background-image: none;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navlinks {
+    display: flex;
+  }
+
+  .homesection {
+    margin-bottom: 60px;
+    flex-wrap: nowrap;
+    height: 100vh;
+  }
+
+  .homewrp,
+  .aboutwrpr {
+    width: 50%;
+    padding-left: 70px;
+    justify-content: flex-start;
+    align-items: flex-start;
+  }
+
+  .maintitle {
+    width: 100%;
+  }
+
+  .mtitle {
+    font-size: 90px;
+    line-height: 90px;
+    padding: 0;
+  }
+
+  .maintxt {
+    width: 90%;
+  }
+
+  .mtext {
+    margin: 0;
+  }
+
+  .decoimg {
+    display: none;
+  }
+
+  .decoimgdsktp {
+    display: block;
+  }
```

Comment:
Your background image has not been positioned the way it is positioned in the template design.

Here's your background:

<img width="688" alt="Screen Shot 2022-06-03 at 13 07 25" src="https://user-images.githubusercontent.com/82426682/171921598-941653c9-c872-41d7-a6e3-ddf599a5321b.png">

Here's the template design background:

<img width="405" alt="Screen Shot 2022-06-03 at 13 08 14" src="https://user-images.githubusercontent.com/82426682/171921710-e5cc433a-96fe-4159-9b20-39576ede2a18.png">

Kindly fix it.